.travis.yml

```diff
@@ -5,5 +5,5 @@ jdk:
   - oraclejdk8
 
 sbt_args: -no-colors
-script: sbt ++$TRAVIS_SCALA_VERSION scalastyle clean coverage test unidoc
```

Comment:
One more thing - I am totally puzzled by this: `++$TRAVIS_SCALA_VERSION`. Why would we want Scala version to be picked from travis (while we can have it in sbt independent of OS wide scala version)